inc/ThirdParty/Hostings/Kinsta.php

```diff
@@ -0,0 +1,193 @@
+<?php
```

Comment:
In this file:
- You can use the `WP_Rocket\ThirdParty\ReturnTypesTrait` methods to return  `false` or an empty array where necessary without creating new methods
- You can remove all `@author` references
- You can remove all `rocket_` prefixes from the methods names